flink-streaming-java/src/main/java/org/apache/flink/streaming/api/environment/ExecutionCheckpointingOptions.java

```diff
@@ -162,10 +162,11 @@
 					"will timeout and checkpoint barrier will start working as unaligned checkpoint.")
 				.build());
 
+	@Documentation.ExcludeFromDocumentation("Do not advertise this option until rescaling of unaligned checkpoint is completed.")
 	public static final ConfigOption<Boolean> FORCE_UNALIGNED =
 		ConfigOptions.key("execution.checkpointing.unaligned.forced")
 			.booleanType()
-			.defaultValue(false)
+			.defaultValue(true)
```

Comment:
Why is this change needed?

Reply:
Without this change UC wouldn't work with iterations. Now that there is no `EndOfChannelStateEvent` and with it no blocking, we can also support iterations for the time being.
Alternatively, we could keep the force option and discourage using UC in iterations already in 1.12.

Reply:
I see, thanks.